fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java

```diff
@@ -70,9 +69,9 @@ private long asEpoch(String time) {
    */
   @GetMapping(params = {"patient"})
   public FallRiskAssessmentResponse searchByPatient(@RequestParam("patient") String patientIcn) {
-    return surveyRepository
-        .findByPatientFullIcnAndSurveyName(patientIcn, FALL_RISK_SURVEY_NAME)
-        .asDatamartSurvey()
+    return fallRiskRepository
+        .findByPatientFullIcn(patientIcn)
+        .asDatamartFallRisk()
```

Comment:
What if this query returns more than 1 record? If so, this gon' break.

Reply:
Now returns list and mapped back.